source/dialogs/drun.c

```diff
@@ -547,6 +556,60 @@ static void drun_icon_fetch ( gpointer data, gpointer user_data )
     rofi_view_reload ();
 }
 
+/**
+ * Initialize desktop entry fields.
+ */
+static void drun_collect_entry_fields ( DRunModePrivateData *pd )
+{
+    unsigned int entry_fields_length = 4;
+    pd->matching_entry_fields                     = g_realloc ( pd->matching_entry_fields, entry_fields_length * sizeof ( DRunEntryField ) );
+    pd->matching_entry_fields[0].entry_field_name = "name";
```

Comment:
Please create an enum for the index, this helps readability. (esp. in the later part)
something like:
```c
typedef enum {
DRUN_MATCH_FIELD_NAME,
DRUN_MATCH_FIELD_EXEC,
DRUN_MATCH_FIELD_CATEGORIES,
...,
DRUN_MATCH_NUM_FIELDS
}  DRunMatchingFields;
```